src/components/SimpleContextSelect.js

```diff
@@ -1,42 +1,59 @@
 import React from 'react'
 import mapDeep from 'deepdash/es/mapDeep'
 import filterDeep from 'deepdash/es/filterDeep'
+import DeleteButton from '../routes/create/components/DeleteButton'
+import styled from 'styled-components'
 
 import config from '../config.json'
-import { useTranslation } from 'react-i18next'
+import { Loading } from './loading'
 
-function SimpleContextSelect ({ onItemChosen, selectedContext, struktur, disabled }) {
-  const { t } = useTranslation('context')
+const RemovableLiElement = styled.li`
+list-style: none;
+height: 2em;
+margin-bottom: calc(var(--margin)/2);
+`
+function SimpleContextSelect ({ onItemChosen, selectedContext, contexts, struktur, disabled, loading, handleRemove }) {
   const items = config.medienhaus?.sites?.moderate?.manageContexts?.showRoot ? struktur : struktur[Object.keys(struktur)[0]].children
+
   return (
     <>
-      <select disabled={disabled} defaultValue={selectedContext} onChange={(e) => { onItemChosen(JSON.parse(e.target.value)) }}>
-        <option disabled selected value="">-- {t('select context')} --</option>
-        {mapDeep(filterDeep(items, (value, key, parent, context) => {
-          // Exclude all hierarchy elements that are not "contexts"
-          if (!value?.type.includes('context')) return false
-          value.path = []
-          value.pathIds = []
-          function addParentToPath (item) {
-            if (item.parentItem.value.name) {
-              value.path.unshift(item.parentItem.value.name)
-              value.pathIds.unshift(item.parentItem.value.id)
-              // Recursion: If this parent has yet another parent item, go check that out
-              if (item.parentItem.parentItem) {
-                addParentToPath(item.parentItem)
+      {contexts?.map((context, index) => {
+        return (
+          <RemovableLiElement key={context.room_id}>
```

Comment:
Here I'd like to make some bigger conceptual suggestions:
The `SimpleContextSelect` component should - as its name says - only be responsible for showing a `<select>` component with all available contexts inside of it. Therefore this list of currently selected contexts does not belong in here.

This would also make us not having to forward all these new props like `handleRemove`, `contexts` and `loading` to `SimpleContextSelect`.

Showing currently selected contexts should then probably live in the parent component, in this case `Category`, where you already - correctly - store the state of `contexts`: https://github.com/medienhaus/medienhaus-cms/pull/142/files#diff-5a04c99c2021f22fa7179a771ddaf43e7832242ed8b3bfe3f7a2e60c28bacbcdR10

That component (`Category`) alone should bother with:
1. listing currently selected contexts (which is this list where I'm currently commenting here)
2. handling removing selected contexts
3. and then show a selector component at the end of the list to allow adding the project to further contexts